auth/src/main/java/org/aerogear/android/ags/auth/credentials/IIntegrityCheckParameters.java

```diff
@@ -0,0 +1,10 @@
+package org.aerogear.android.ags.auth.credentials;
+
+public interface IIntegrityCheckParameters {
+
+    String getAudience();
+    String getIssuer();
+    String getPublicKey();
+    boolean isValid();
+    String serialize();
```

Comment:
Do you think JavaDocs are needed?  The properties and things look self explanatory to me, but if there are any weirdnesses or edge cases to be aware of (like what does isValid check to validate) then I would want them added.

Reply:
I think you're right about isValid requiring some docs here. For the others I don't think so, they're self explanatory I think.